tools/ds-identify

```diff
@@ -179,10 +179,26 @@ debug() {
     echo "$@" 1>&3
 }
 
+map_sysfs_to_dmi() {
+    case "$@" in
+        sys_vendor) _RET="system-manufacturer";;
+        product_name) _RET="system-product-name";;
+        product_uuid) _RET="system-uuid";;
+        product_serial) _RET="system-serial-number";;
+        *) _RET=$(echo "$@" | tr '_' '-');;
```

Comment:
just blow up on unknown key.  this attempt to do the right thing will just make failures harder to find.

Reply:
for now, there's exactly one key that fits this pattern: `chassis_asset_tag`.

How would you blow up? `return 1`?

Reply:
oh bummer. we are not checking anything there.
I guess what it is doing right now is *only* setting _RET=$ERROR.

maybe:

    _RET="$ERROR: unknown field XXXXX"